bin/bump-version

```diff
@@ -49,35 +49,34 @@ fi
 
 if $console_bump; then
     console_dir=console
+    rm -rf $console_dir
     if is_truthy "${CI:-0}"; then
         GIT_TERMINAL_PROMPT=0 git clone "https://materializebot:$GITHUB_TOKEN@github.com/MaterializeInc/console" $console_dir
     else
         GIT_TERMINAL_PROMPT=0 git clone git@github.com:MaterializeInc/console $console_dir
     fi
-    echo "Bumping console version to $version"
+    echo "Bumping console version to v$version"
     (
         cd "$console_dir"
-        git tag -a "$version" -m "$version"
-        git push origin "$version"
+        git tag -a "v$version" -m "v$version"
+        git push origin "v$version"
     )
 
     echo "Waiting for console version to be released on DockerHub (~15 min)"
-    console_version="${version#v}"
-    console_image="materialize/console:$console_version"
-    sleep $((15 * 60))
+    console_image="materialize/console:$version"
     while true; do
         if docker manifest inspect "$console_image" >/dev/null 2>&1; then
             break
         fi
-        echo "$console_image not yet on DockerHub, sleeping 1 min"
-        sleep 60
+        echo "$console_image not yet on DockerHub, sleeping 5 min"
```

Comment:
Is a 5x reduction in `sleeping` lines worth waiting up to 4 extra minutes unnecessarily?

How about separating these two aspects: only echo every 5 mins, but sleep in 1 min units? Of even 15 seconds?